src/hooks/use-import-export.tsx

```diff
@@ -84,15 +84,24 @@ export const ImportExportProvider = ( { children }: { children: React.ReactNode
 				},
 			} ) );
 
-			const handleImportError = async ( error?: unknown ) => {
-				await getIpcApi().showErrorMessageBox( {
-					title: __( 'Failed importing site' ),
-					message: __(
-						'An error occurred while importing the site. Verify the file is a valid Jetpack backup, Local, Playground, .wpress or .sql database file and try again. If this problem persists, please contact support.'
-					),
-					error,
-					showOpenLogs: true,
-				} );
+			const handleImportError = async ( error: unknown ) => {
+				if ( error instanceof Error && error.message.includes( 'absolute path' ) ) {
```

Comment:
Or is this better?

```suggestion
				if ( error instanceof Error && error.message.includes( 'Error: absolute path: /' ) ) {
```

Reply:
I think following your updated suggestion makes sense 👍 `"absolute path"` is relatively ambiguous.

It'd be nice if yauzl provided an error code, but it doesn't…

Reply:
Yep, this will have to do for now.